tools/summary-report/environment.yml

```diff
@@ -0,0 +1,8 @@
+name: report-summary
+channels:
+  - anaconda
+  - conda-forge
+dependencies:
+  - conda-forge::python=3.6
```

Comment:
Surely we can put this software into the base container? Would be nice to have at least one fewer containers if we can..

Reply:
WIP

Reply:
Done